### DISCUSSION TITLE
Git Actions

### DISCUSSION THREAD

**Discussion by egecans** [bounswe/bounswe2023group2#159] [Ideas]
We will use git in Milestone 2; thus, we should get familiar with git actions. Maybe some of us know them very properly but some few don't know them as well as they do. Therefore we can share useful links below this discussion. We can ask if is there any more git action to we need to learn?

**Comment by egecans** [comment]
Let's start with this question. For now, I know and use git commit, push, merge requests, compare, branches and check out. Is there anything that I should learn more about for this project?

**Comment by bgmrsln** [comment]
I think we can start watching this [video](https://www.youtube.com/watch?v=1ffBJ4sVUb4&t=117s) that lecturer shared.

**Reply by egecans** [reply]
That video would be useful if we were in the 2010s :D
This [video](https://www.youtube.com/watch?v=tRZGeaHPoaw) is more ok than that I think
